test/unit/auth_test.js

```diff
@@ -28,16 +28,41 @@ describe('building authorization entries', function () {
     );
 
     let cred = entry.credentials().address();
-    let args = cred.signatureArgs().map((v) => StellarBase.scValToNative(v));
-
     expect(cred.signatureExpirationLedger()).to.equal(123);
-    expect(args.length).to.equal(1);
+    expect(entry.rootInvocation()).to.eql(invocation);
+
+    // sanity check raw xdr signature types: should be ScVal{
```

Comment:
not suggesting any change, since mostly defined the expected outcome as xdr js object here, would using that as `expect(sig).to.equal(expectedXdrSig)` be a substitute instead of multiple lower level verification checks, and then the code becomes the docs rather than maintaining over time, which is great that it's here in any case for those less in tune like myself.